### PR TITLE
fr&de&cn&ja translations for ‘beforeSeconds’.

### PR DESCRIPTION
**Pull request by ShadyWhite**

### CODE REVIEW

**Comment by Akurosia** [comment]
could you add german as well `de: 'Vorher (Sekunden)',` ?

File: ui/raidboss/raidboss_config.js [hunk 1]
```diff
@@ -115,6 +115,8 @@ const kDetailKeys = {
   'beforeSeconds': {
     label: {
       en: 'before (sec)',
+      ja: 'その前に（秒）',
+      cn: '提前（秒）',
```
Comment (MaikoTan) [inline ui/raidboss/raidboss_config.js:119]: no, use 半角 instead of 全角, all of this.
just remember to add a space before it.

**Review by MaikoTan — CHANGES_REQUESTED** [review]

**Comment by ShadyWhite** [comment]
> could you add german as well `de: 'Vorher (Sekunden)',` ?

added.

**Comment by MaikoTan** [comment]
BTW, français may be `'avant (seconde)'`
But I am not very confident with it.

**Pull request merged**